src/loans/kamino.py

```diff
@@ -257,4 +257,112 @@ def compute_liquidable_debt_at_price(
+        for token
+        in collateral_tokens
+    }
+    debt_token_parameters = {
+        debt_token: src.kamino_vault_map.supply_to_info_map.get(debt_token, None)
+        for debt_token
+        in debt_tokens
+    }
+
+    for token in collateral_tokens:
+        loan_states[f'collateral_{token}'] = loan_states['collateral'].apply(
+            lambda x: x[token] if token in x else decimal.Decimal('0')
+        )
+    for token in debt_tokens:
+        loan_states[f'debt_{token}'] = loan_states['debt'].apply(
+            lambda x: x[token] if token in x else decimal.Decimal('0')
+        )
+    
+    for token in collateral_tokens:
+        if not collateral_token_parameters[token]:
+            continue
+        if not collateral_token_parameters[token]['underlying_decs']:
+            continue
+        decimals = collateral_token_parameters[token]['underlying_decs']
+        ltv = collateral_token_parameters[token]['ltv']
+        underlying_token = src.kamino_vault_map.lp_to_mint_map[token]
+
+        loan_states[f'collateral_usd_{token}'] = (
+            loan_states[f'collateral_{token}'].astype(float)
+            / (10**decimals)
+            * token_prices[underlying_token]
+        )
+        
+        loan_states[f'risk_adj_collateral_usd_{token}'] = (
+            loan_states[f'collateral_{token}'].astype(float)
+            / (10**decimals)
+            * (ltv/100)
+            * token_prices[underlying_token]
+        )
+    
+    for debt_token in debt_tokens:
+        if not debt_token_parameters[debt_token]:
+            continue
+        if not debt_token_parameters[debt_token]['underlying_decs']:
+            continue
+        decimals = debt_token_parameters[debt_token]['underlying_decs']
+        ltv = debt_token_parameters[debt_token]['ltv']
+        underlying_token = src.kamino_vault_map.supply_vault_to_mint_map[debt_token]
+        loan_states[f'debt_usd_{debt_token}'] = (
+            loan_states[f'debt_{debt_token}'].astype(float)
+            / (10**decimals)
+            * token_prices[underlying_token]
+        )
+
+        loan_states[f'risk_adj_debt_usd_{debt_token}'] = (
+            loan_states[f'debt_{debt_token}'].astype(float)
+            / (10**decimals)
+            * (1/(ltv/100) if ltv else 1)
+            * token_prices[underlying_token]
+        )
+    loan_states['risk_adj_debt_usd'] = loan_states[[i for i in loan_states.columns if i.startswith('risk_adj_debt_usd_')]].sum(axis=1)
+    loan_states['debt_usd'] = loan_states[[i for i in loan_states.columns if i.startswith('debt_usd_')]].sum(axis=1)
+
+    loan_states['risk_adj_collateral_usd'] = loan_states[[i for i in loan_states.columns if i.startswith('risk_adj_collateral_usd_')]].sum(axis=1)
+    loan_states['collateral_usd'] = loan_states[[i for i in loan_states.columns if i.startswith('collateral_usd_')]].sum(axis=1)
+
+    loan_states['std_health'] = (loan_states['risk_adj_collateral_usd'] / loan_states['risk_adj_debt_usd']).fillna(np.inf)
```

Comment:
Fillna not necessary. NAs are fine.

Reply:
The the sort_values method would hate it wouldn't it?